plugins/pipelines/update.js

```diff
@@ -6,6 +6,7 @@ const schema = require('screwdriver-data-schema');
 const idSchema = schema.models.pipeline.base.extract('id');
 const { formatCheckoutUrl, sanitizeRootDir } = require('./helper');
 const { getUserPermissions } = require('../helper');
+const ANNOTATION_USE_DEPLOY_KEY = 'screwdriver.cd/useDeployKey';
```

Comment:
Wondering if it would be better to put these in a common file somewhere or leverage data-schema
but we can do that in a separate PR